gtk4/src/functions.rs

```diff
@@ -25,3 +25,12 @@ pub fn show_about_dialog<P: IsA<Window>>(parent: Option<&P>, properties: &[(&str
     about_dialog.set_transient_for(parent);
     about_dialog.show();
 }
+
+#[doc(alias = "gtk_test_list_all_types")]
```

Comment:
I suppose this can't be generated automatically due to the `zero-terminated=1` annotation; although, not sure if it's really the case. see https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gtk/gtktestutils.c#L152-158

Reply:
It's probably terminated with `G_TYPE_INVALID`, i.e. `0`.

But if there's a length parameter we should simply ignore the `zero-terminated` annotation